index.d.ts

```diff
@@ -0,0 +1 @@
+export * from './dist';
```

Comment:
Wouldn't you want to export src here? I'm guessing you want to export the TS files from here for your IDE right? This is the line that's breaking the tests and I'm guessing that's because the build can't build because the build hasn't been built yet so this dir doesn't exists when building.

So I'm wondering if this change should be excluded from the tsconfig file or whether we need it? Do you have some documentation on the IDE's autocompletion root file I can look at?

Reply:
Yes, you're right.
I think that there is need dist folder, because when you add the package to the project, it is already built and all files should be imported from dist folder.

For example https://github.com/nestjs/typeorm has root index file and it exports all from 'dist' folder.

As for IDE - I'm using [PhpStorm](https://www.jetbrains.com/phpstorm/).
When I want to import, for example `ConfigService` from the package PhpStorm's autocompletion adds `'nestjs-config/dist'` as import path. For the `ConfigParam` autocompletion adds `''nestjs-config/dist/decorators/ConfigParam''`

I'll try to check PR again today

Reply:
ahhh ok, I'll look into how to exclude a file from the tsconfig or I'll look at typeorm's tsconfig :) thanks!